web/src/Web.App/Views/SchoolSpendingComparison/Index.cshtml

```diff
@@ -11,6 +12,35 @@
         kind = OrganisationTypes.School
     })
 
+<div class="govuk-grid-row govuk-!-margin-bottom-5">
+    <div class="govuk-grid-column-full">
+        <ul class="app-headline app-headline-3 govuk-!-text-align-centre">
+            <li class="app-headline-high">
+                <p class="govuk-body govuk-!-font-size-48 govuk-!-margin-bottom-1">@Model.HighHeadline.CustomCount @Model.HighHeadline.Change</p>
```

Comment:
chore: `<span class="govuk-visually-hidden">` somewhere for accessibility support on 'change' icon